### ISSUE TITLE
Sip Express Media Server (1.7-dev-90-g97d29b0 (x86_64/linux)) - SBC cannot transcode

### ISSUE THREAD

**Issue by linhtn121** [sems-server/sems#44]
[Hi you, we want to use SEMS to transcode but it's not working - 

Caller understands only PCMA codec, callee: understands only PCMU codec:

**my.sbcprofile.conf:**

enable_rtprelay=yes
enable_transcoder=always
transcoder_codecs=pcmu,pcma
prefer_existing_codecs=yes
codec_preference=pcma/8000,pcmu/8000
sdp_filter=whitelist
sdpfilter_list=pcma

The sip signaling is  ok, but the media is not working....

The log: http://pastebin.com/6swu0hfG

What's wrong in my config? Please help us! 

Thank you!

**Comment by nghiacua** [comment]
Dear linhtn121
Please check plugin wav.so? It's loaded or not?

**Comment by linhtn121** [comment]
Dear nghiacua
It's loaded!
I have a debug log when the SEMS loaded: http://pastebin.com/ttCp9Q5b

**Comment by nghiacua** [comment]
Everything's seemly OK!
Did you add "my.sbcprofile.conf" into /usr/locale/etc/sems/etc/sbc.conf file?

**Comment by linhtn121** [comment]
No, my.sbcprofile.conf config is different sbc.conf file, It is just same location

**Comment by nghiacua** [comment]
What is the error in Sip app when you using with and without your config? 
When you remove your config, please check the log file to confirm the transcoder is disabled.

**Comment by linhtn121** [comment]
**Server side:** there are not any error when I using with and without my config.
**Client side:**
**When using with my config:(Caller  using PCMA, callee using PCMU)** 
- Linphone IOS - crash app. Lingphone Desktop: upload ok, download = 0kb;
**Using without myconfig:**
-client show: "not acceptable here'

Could you show me a sample for transcode?

**Comment by nghiacua** [comment]
If you use linephone desktop + without your config + same codecs(PCMA): It is ok, right?
I use to use zoiper app to make call. Could you please try it?

**Comment by linhtn121** [comment]
If caller and callee using the same codecs (PCMA or PCMU) : everything is OK!

**Comment by nghiacua** [comment]
Oh, I have not found any wrong in your configs?
Please try to use wireshark (or tcpdump) to check media stream in sems server?
"Lingphone Desktop: upload ok, download = 0kb"? Do you know "upload" and "download" mean?
Upload: meadia stream from linephone -> sems
Download: media stream from sems -> linephone, right?

**Comment by linhtn121** [comment]
`"Lingphone Desktop: upload ok, download = 0kb"? Do you know "upload" and "download" mean?
Upload: meadia stream from linephone -> sems
Download: media stream from sems -> linephone, right?`
- > yes!

Do you have any simple config for transcoding?
